streams/cluster/src/main/java/com/hortonworks/streamline/streams/cluster/service/metadata/HBaseMetadataService.java

```diff
@@ -153,7 +152,8 @@ public static HBaseMetadataService newInstance(Configuration hbaseConfig,
             return new HBaseMetadataService(ConnectionFactory.createConnection(hbaseConfig, user)
                     .getAdmin(), securityContext, subject, user, hbaseMaster, hbaseMasterProcesses);
         } else {
-            return newInstance(hbaseConfig);
+            return new HBaseMetadataService(ConnectionFactory.createConnection(hbaseConfig).getAdmin(),
```

Comment:
Why not passing securityContext and subject given that it is not fetched from this method? Looks like only `user` is guard against null.

And I'd like to remove 1 arg constructor as well and remove all unused static newInstance() so that we don't make same mistake again. May also need to modify test code though.